packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -985,7 +1103,8 @@ export function getSecurityRequirements(
   path: string,
   method: string,
   securitySchemes: { [key: string]: ProcessedSecurityScheme },
-  oas: Oas3
+  oas: Oas3,
+  callback?: CallbackObject
```

Comment:
Added an optionnal CallbackObject to be able to reuse frunctions provided by oas_3_tools